docs/querying/sql-functions.md

```diff
@@ -1282,21 +1282,74 @@ Returns the following:
 
 ## BLOOM_FILTER
 
-Computes a Bloom filter from values produced by the specified expression.
+Computes a [bloom filter](../development/extensions-core/bloom-filter.md) from values provided in an expression.
 
-* **Syntax**: `BLOOM_FILTER(expr, <NUMERIC>)`
+`numEntries` specifies the maximum number of distinct values before the false positive rate increases.
+
+* **Syntax:** `BLOOM_FILTER(expr, numEntries)`
 * **Function type:** Aggregation
 
+<details><summary>Example</summary>
+
+The following example returns a base64-encoded bloom filter string for entries in `agent_category`:
+
+```sql
+SELECT
+  agent_category,
+  BLOOM_FILTER(agent_category, 10) as bloom
+FROM "kttm"
+  GROUP BY agent_category
+```
+
+Returns the following:
+
+| `agent_keys` | `bloom` |
+| -- | -- |
+| _`empty`_ | `"BAAAAAgAAAAAABAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEABAAAAAA"` |
```

Comment:
We might want to consider another way of formatting this (remove italics?) or see if it can be added to the spelling file. This line and the other instance of _`empty`_ are raising spelling errors.

Reply:
Removed the italics.